src/SamplesApp/SamplesApp.UITests/MessageDialogTests/MessageDialogTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.MessageDialogTests
+{
+	[TestFixture]
+	public partial class MessageDialogTest : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void When_Click_Outside_Dialog_Expect_No_Dismiss()
+		{
+			Run("UITests.MessageDialog.MessageDialogTest");
+			var button = _app.Marked("showDialogBtn");
+			_app.WaitForElement(button);
+			button.Tap();
```

Comment:
Use `.FastTap()` (it just takes less time to process and makes the tests run faster).